README.md

```diff
@@ -64,6 +64,13 @@ The `ghcide` executable mostly relies on [`hie-bios`](https://github.com/mpicker
 
 If you can't get `ghcide` working outside the editor, see [this setup troubleshooting guide](docs/Setup.md). Once you have got `ghcide` working outside the editor, the next step is to pick which editor to integrate with.
 
+### Optimal project setup
+
+`ghcide` has been designed to handle projects with hundreds or thousands of modules. If `ghci` can handle it, then `ghcide` should be able to handle it too provided the following conditions are met:
+
+1. Use GHC >= 8.6
+2. Give `ghcide` a chance to load all the project modules and generate interface files. This happens automatically for all the dependencies of the modules being edited, but may take a while and require a restart to free up resources after complete.
```

Comment:
This could be clearer - how do I "give `ghcide` a chance" to load everything? Will it automatically find everything itself, or do I have to tell it do something? Do I do this from my editor, or from the command line? Also, you mention it will take a while - how will I know when it's done?

Reply:
You are right, this is not very helpful. Since everything happens automatically and (Info) diagnostics are generated with the message `additional resource use while generating interface files in the background` while this is being done, I think we can safely remove the second item in the list. The only caveat is that trying to benchmark ghcide memory usage on a single run is not going to be accurate, since it will be slower / use more memory due to the generation of `.hi`/`.hie` files for project modules.

Reply:
How about:
```

`ghcide` has been designed to handle projects with hundreds or thousands of modules. If `ghci` can handle it, then `ghcide` should be able to handle it. The only caveat is that this currently requires GHC >= 8.6, and that the first time a module is loaded in the editor will trigger generation of support files in the background if those do not already exist.
```

Reply:
The latter is much better imo. It tells me what is happening, but doesn't ask me to do anything in particular

Reply:
The latter is much better imo. It tells me what is happening, but doesn't ask me to do anything in particular

Reply:
Not sure if all `8.6.*` will support hie.files, if it is not the case we should note the specific 8.6 version that does it.